src/pmacfiltercontrol/pmacFilterControlWrapper.py

```diff
@@ -862,7 +862,9 @@ def _set_shutter_pos(self, val: float, shutter_state: str) -> None:
         if shutter_state == SHUTTER_CLOSED:
             self._configure_param({"shutter_closed_position": val})
 
-        current_shutter_state = "CLOSED" if self.shutter.get() == 0 else "OPEN"
+        current_shutter_state = (
+            SHUTTER_CLOSED if self.shutter.get() == 0 else SHUTTER_OPEN
+        )
         if current_shutter_state == shutter_state:
             self._set_shutter(shutter_state)
```

Comment:
Isn't this wrong? Shouldn't it only call set if the current value does _not_ match the new value?